samples/v2/pipeline_with_volume.py

```diff
@@ -62,7 +62,6 @@ def pipeline_with_volume():
 
 if __name__ == '__main__':
     # execute only if run as a script
-    from kfp import compiler
     compiler.Compiler().compile(
         pipeline_func=pipeline_with_volume,
-        package_path='pipeline_with_volume.json')
\ No newline at end of file
+        package_path='pipeline_with_volume.yaml')
```

Comment:
Make sure to add a newline EoF :)

Reply:
Man!!! I will add it on the next commit when I add several more IR files here.